Postman/index.js

```diff
@@ -1,27 +1,21 @@
-"use strict";
+require('babel/register');
 
-var Cronjob = require('cron').CronJob;
-var moment = require('moment-timezone');
+const Cronjob = require('cron').CronJob;
+const Postman = require('Postman/Postman')();
+const runPostSeries = require('Postman/PostSeries').runPostSeries;
 
-var Postman = require('Postman/Postman')();
+const morning = '0 0 21 * * *';
```

Comment:
__[jshint]__
*  'const' is available in ES6 (use esnext option) or Mozilla JS extensions (use moz).